collector/start_service/start_service.go

```diff
@@ -0,0 +1,43 @@
+package start_service
+
+import (
+	"github.com/hooklift/gowsdl/soap"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/SUSE/sap_host_exporter/collector"
+	"github.com/SUSE/sap_host_exporter/internal/sapcontrol"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func NewCollector(startServiceUrl string) (*soapCollector, error) {
+	webService := sapcontrol.NewWebService(soap.NewClient(startServiceUrl))
+
+	c := &soapCollector{
+		collector.NewDefaultCollector("start_svc"),
```

Comment:
Imho service is better the. Svc

Reply:
also meaning that this namespace will showed in metric imho is not clear what svc is

Reply:
yeah, you're totally right!